tests/src/lib.rs

```diff
@@ -0,0 +1 @@
+// Intentionally empty
```

Comment:
Perhaps you want to add a comment like that in `workspaces/icecap-host/src/icecap-host/src/lib.rs`, too.

(I'm commenting here because I can't see how to add a comment on an empty file!)

Reply:
Also `workspaces/linux-host/src/linux-host/src/lib.rs`.